<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YT-DLP-QT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="url">
        <property name="placeholderText">
         <string>Paste URL to download here</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_format">
       <attribute name="title">
        <string>Format</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="group_video">
          <property name="title">
           <string>Video</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lbl_resolution">
               <property name="text">
                <string>Max. Resolution</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="video_resolution">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>4k</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1440p</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1080p</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>720p</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>480p</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lbl_video_format">
               <property name="text">
                <string>Preferred Format</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="video_format">
               <item>
                <property name="text">
                 <string>MP4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MKV</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3GP</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>FLV</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WEBM</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="group_audio">
          <property name="title">
           <string>Audio</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="audio_only">
             <property name="text">
              <string>Audio Only</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lbl_quality">
               <property name="text">
                <string>Quality</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="audio_quality">
               <item>
                <property name="text">
                 <string>Best</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Good</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Medium</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Low</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lbl_audio_format">
               <property name="text">
                <string>Format</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="audio_format">
               <item>
                <property name="text">
                 <string>AAC</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>OGG</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>MP3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>OPUS</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>WAV</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progress">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="monitor_clipboard">
          <property name="text">
           <string>Get URL from clipboard (experimental)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="lbl_output_name">
            <property name="text">
             <string>Output Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="output_name">
            <property name="text">
             <string>%(title)s.%(ext)s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="lbl_own_format">
            <property name="text">
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="own_format">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>159</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="browse">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_path"/>
      </item>
      <item>
       <widget class="QPushButton" name="download">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>436</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
